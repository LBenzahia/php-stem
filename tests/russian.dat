а => а
абиссинию => абиссин
абонемента => абонемент
абонировался => абонирова
абонируйся => абонир
абонируюсь => абонир
абрикосы => абрикос
август => август
августа => август
августе => август
авдотье => авдот
авдотьей => авдот
авдотьи => авдот
авдотью => авдот
авдотья => авдот
авенантненькая => авенантненьк
аверка => аверк
аверкой => аверк
авиатор => авиатор
авиаторов => авиатор
авиация => авиац
авось => ав
авраама => авраам
австрии => австр
австрийский => австрийск
автобиографию => автобиограф
автографом => автограф
автомобили => автомоб
автомобиль => автомобил
автор => автор
авторам => автор
авторитет => авторитет
авторитета => авторитет
авторитеты => авторитет
авторов => автор
автором => автор
ага => аг
агадир => агадир
агамемнон => агамемнон
агаповну => агаповн
агафье => агаф
агафьей => агаф
агафьи => агаф
агафью => агаф
агафья => агаф
агента => агент
агонии => агон
агроном => агрон
агрономии => агроном
ад => ад
ада => ад
адам => ад
адамант => адамант
адвокат => адвокат
адвокатом => адвокат
адвокатского => адвокатск
адвокатской => адвокатск
адвокатскую => адвокатск
административный => административн
административных => административн
администратор => администратор
адмиралы => адмирал
адом => ад
адрес => адрес
адреса => адрес
адресный => адресн
адресовано => адресова
адресом => адрес
адресу => адрес
адрианополя => адрианопол
адриатической => адриатическ