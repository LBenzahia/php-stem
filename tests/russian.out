–∞
–∞–±–∏—Å—Å–∏–Ω
–∞–±–æ–Ω–µ–º–µ–Ω—Ç
–∞–±–æ–Ω–∏—Ä–æ–≤–∞
–∞–±–æ–Ω–∏—Ä
–∞–±–æ–Ω–∏—Ä
–∞–±—Ä–∏–∫–æ—Å
–∞–≤–≥—É—Å—Ç
–∞–≤–≥—É—Å—Ç
–∞–≤–≥—É—Å—Ç
–∞–≤–¥–æ—Ç
–∞–≤–¥–æ—Ç
–∞–≤–¥–æ—Ç
–∞–≤–¥–æ—Ç
–∞–≤–¥–æ—Ç
–∞–≤–µ–Ω–∞–Ω—Ç–Ω–µ–Ω—å–∫
–∞–≤–µ—Ä–∫
–∞–≤–µ—Ä–∫
–∞–≤–∏–∞—Ç–æ—Ä
–∞–≤–∏–∞—Ç–æ—Ä
–∞–≤–∏–∞—Ü
–∞–≤
–∞–≤—Ä–∞–∞–º
–∞–≤—Å—Ç—Ä
–∞–≤—Å—Ç—Ä–∏–π—Å–∫
–∞–≤—Ç–æ–±–∏–æ–≥—Ä–∞—Ñ
–∞–≤—Ç–æ–≥—Ä–∞—Ñ
–∞–≤—Ç–æ–º–æ–±
–∞–≤—Ç–æ–º–æ–±–∏–ª
–∞–≤—Ç–æ—Ä
–∞–≤—Ç–æ—Ä
–∞–≤—Ç–æ—Ä–∏—Ç–µ—Ç
–∞–≤—Ç–æ—Ä–∏—Ç–µ—Ç
–∞–≤—Ç–æ—Ä–∏—Ç–µ—Ç
–∞–≤—Ç–æ—Ä
–∞–≤—Ç–æ—Ä
–∞–≥
–∞–≥–∞–¥–∏—Ä
–∞–≥–∞–º–µ–º–Ω–æ–Ω
–∞–≥–∞–ø–æ–≤–Ω
–∞–≥–∞—Ñ
–∞–≥–∞—Ñ
–∞–≥–∞—Ñ
–∞–≥–∞—Ñ
–∞–≥–∞—Ñ
–∞–≥–µ–Ω—Ç
–∞–≥–æ–Ω
–∞–≥—Ä–æ–Ω
–∞–≥—Ä–æ–Ω–æ–º
–∞–¥
–∞–¥
–∞–¥
–∞–¥–∞–º–∞–Ω—Ç
–∞–¥–≤–æ–∫–∞—Ç
–∞–¥–≤–æ–∫–∞—Ç
–∞–¥–≤–æ–∫–∞—Ç—Å–∫
–∞–¥–≤–æ–∫–∞—Ç—Å–∫
–∞–¥–≤–æ–∫–∞—Ç—Å–∫
–∞–¥–º–∏–Ω–∏—Å—Ç—Ä–∞—Ç–∏–≤–Ω
–∞–¥–º–∏–Ω–∏—Å—Ç—Ä–∞—Ç–∏–≤–Ω
–∞–¥–º–∏–Ω–∏—Å—Ç—Ä–∞—Ç–æ—Ä
–∞–¥–º–∏—Ä–∞–ª
–∞–¥
–∞–¥—Ä–µ—Å
–∞–¥—Ä–µ—Å
–∞–¥—Ä–µ—Å–Ω
–∞–¥—Ä–µ—Å–æ–≤–∞
–∞–¥—Ä–µ—Å
–∞–¥—Ä–µ—Å
–∞–¥—Ä–∏–∞–Ω–æ–ø–æ–ª
–∞–¥—Ä–∏–∞—Ç–∏—á–µ—Å–∫
–∞–¥—Å–∫
–∞–¥—Å–∫
–∞–∂–Ω
–∞–∑–∞—Ä—Ç
–∞–∑–∞—Ä—Ç
–∞–∑–±—É–∫
–∞–∑–≤–æ–ª—å—Ç
–∞–∑–∏–∞—Ç—Å–∫
–∞–∑–∏–∞—Ç—á–∏–Ω
–∞–∑
–∞–∑
–∞–∑—Ä–∞
–∞–∑
–∞
–∞
–∞–π–≤–∞–∑–æ–≤—Å–∫
–∞–π–¥
–∞–∫
–∞–∫–∞–¥–µ–º
–∞–∫–∞–¥–µ–º
–∞–∫–∞—Ü
–∞–∫–∞—Ü
–∞–∫–≤–∞—Ä–µ–ª
–∞–∫–∫–æ–º–ø–∞–Ω–µ–º–µ–Ω—Ç
–∞–∫–∫–æ–º–ø–∞–Ω–∏—Ä–æ–≤–∞
–∞–∫–∫–æ—Ä–¥
–∞–∫–∫–æ—Ä–¥
–∞–∫–∫–æ—Ä–¥
–∞–∫–∫—É—Ä–∞—Ç–Ω
–∞–∫–∫—É—Ä–∞—Ç–Ω
–∞–∫–∫—É—Ä–∞—Ç–Ω
–∞–∫–∫—É—Ä–∞—Ç–Ω
–∞–∫–º–µ–∏–∑–º
–∞–∫—Å–µ–ª—å–±–∞–Ω—Ç
–∞–∫—Å–∏–Ω
–∞–∫—Å–∏–æ–º
–∞–∫—Ç
–∞–∫—Ç
–∞–∫—Ç
–∞–∫—Ç–µ—Ä
–∞–∫—Ç–µ—Ä
–∞–∫—Ç–µ—Ä
–∞–∫—Ç–µ—Ä
–∞–∫—Ç–∏–≤–Ω
–∞–∫—Ç—Ä–∏—Å
–∞–∫—Ç—Ä–∏—Å
–∞–∫—Ç—Ä–∏—Å
–∞–∫—É–ª–∏–Ω
–∞–∫—É–ª–∏–Ω
–∞–∫—É–ª–∏–Ω
–∞–∫—É–ª–∏–Ω
–∞–∫—É–ª–∏–Ω
–∞–∫—É—Ä–∞—Ç
–∞–∫—É—à–µ—Ä
–∞–∫—Ü–µ–Ω—Ç
–∞–∫—Ü–∏–∑
–∞–∫—Ü
–∞–∫—Ü
–∞–∫—Ü
–∞–∫—Ü
–∞–ª
–∞–ª
–∞–ª–≥–µ–±—Ä
–∞–ª–≥–µ–±—Ä
–∞–ª–≥–µ–±—Ä
–∞–ª–µ–±–∞—Å—Ç—Ä
–∞–ª–µ–∫
–∞–ª–µ–∫—Å–∞–Ω–∫–∞–ª–µ–∫—Å––∞–Ω–∫–∞–µ–∫—Å––∞––Ω
–∞–∫—É–ª–∏–≤—Å–∫
–—Å––∞––Ω
–∞–∫—É–Ä–æ–Ç–∏–∫
–—Å––∞––Ω
–∞–∫—É–Ä–æ–Ω–∫
–—Å––∞––Ω
–∞–∫—É–Ä–æ–Ω–∫
–—Å––∞––Ω
–∞–∫—É–Ä–æ–Ω–∫
–—Å––∞––Ω
–∞–∫—É–Ä–æ–Ω–∫
–—Å––∞––Ω
–∞–∫—É–Ä–æ–Ω–∫
–—Å––∞––Ω
–∞–∫—É–∫
–—Å––∞––Ω
–∞–∫—É–∫
–—Å––∞––Ωµ–æ–Ω–∫
–—Å––∞––Ωµ–∫
–—Å––∞––Ωµ–µ–∞–≤
–—Å––∞––Ωµ–µ–∞–Ç–∏–∫
–—Å––∞––Ωµ–µ–∞–Ç–∏–∫
–—Å––∞––Ωµ–µ–∞–Ç–∏–∫
–—Å––∞––Ωµ–µ–∞–∫
–—Å––∞––Ωµ–µ–∞–∫
–—Å––∞––Ωµ–µ–∞–∫
–—Å––∞––Ωµ–Ç–∏–∫
–—Å––∞––Ωµ–Ç–∏–∫
–—Å––∞–––∫
–—Å––∞––Ωµ–∫
–—Å––∞–ª
–∞–ª
–∞–µ–Ω–∫
–—Å––Ω–ª
–∞–ª
–∞–µ–Ω–Ω—å–∫
–∞–µ—à–∫
–∞–µ—à—å–∫
–∞–µ—é–Ω—Ç
–ªΩµ–∫
–——Ç
–ªΩ——Ç
–ªΩ—–µ–≥—Ç–æ—Ä
–ª–——Ç
–ªΩ——Ç
–ªΩ——Ç
–ªΩ—–µ—Ç
–ªΩ—–µ—Ç
–ªΩº∑
–∞–∑
–ªΩº∑
–∞–∑
–ªΩº∑
–∞–∑
–ªΩº∑
–∞–∑
–ª–∫
–——Ç
–ªΩæ—ç—Ç
–ª–∫—∞–æ—Ä
–ª–∫—∞–æ—Ä
–ª–∫—∞–æ—Ä
–ª–á–∫—â–∑
–ª–∫
–——Ç
–ª—Ç
–ª—Ç
–ª—Ç
–ª—Ç
–ª–ª—å–µ–∞—Ä—ã–∏–∫
–—–ª—å—Ç
–ª–ª—å–∏–æ–º
–ª–ª—æ–º
–ª–ª—Ä—à—∞–É—à–µ—Ä
–ª–ª—è–Ω–ª
–ºÇ
–ª—Ç
–ºÇ
–ª—Ç
–ºÇ
–ª—Ç
–ºÇ
–ª—Ç
–ºÇ
–ª—Ç
–ºÇ±—∞–æ—Ä
–ºÇ±—∞–æ—Ä
–ºÇ±—∞–æ—Ä
–ºÇ±—∞–æ—Ä
–ºÇ±—∏–∫—Ü
–ºÇ±—∏–∫—Ü
–ºÇµ–±—Ä–∏—Ü
–ºÇµ–±—Ä–∏–±–∞–Å–∏—Ü
–ºÇµ–±—Ä–∏—Ü
–ºÇµ–±—Ä–∏—Ü
–ºÇµ–±—Ä–∏—Ü
–ºÇµ–Ç–Ω–∏—Å—Ü
–ºÇ∏–Ω–ª
–ºÇΩ–ª
–Ω–ª
–ΩÇ
–ªΩÜ–∏–∑
–ΩÇ
–ªΩÜ–∏–∑
–ΩÇ
–ªΩÜ–∏–Ω–∏—Ä–æ–≤–∞
–ΩÇ
–ªΩÜ–∏–Ω–∏–ª
–ΩÇ
–ªΩÜ–∏–∑
–ΩÇ
–ªΩÜ–∏–∑
–ΩÇ
–ªΩæ–∑
–ΩÇ
–º–Ω–ª
–ΩÇ
–Ω–±–∞–ª
–ΩÇ
–Ω–±–∞–ª
–ΩÇ
–∏—Å–±–∞–ª
–ΩÇ
–Ç–∏–æ–º
–ΩÇ
–Ç–∏–æ–º
–ΩÇ≥∫
–∞–Ω–µ–º–µ–Ω—Ç
–ΩÇ≥∫
–∏–ª
–ΩÇ≥Å––∞–ª
–ΩÇ≥Å––∞–ª
–ΩÇ≥Å––∞Ç∏–Ω–ª
–ΩÇ≥Å––∞–ª
–ΩÇ≥Å––∞–ª
–ΩÇ≥Å––∞Çæ—Ç—∫–ª
–ΩÇ≥Å––∞–ª
–ΩÇ≥Å––∞–ª
–ΩÇ≥Å––∞–ª—Å—∫–ª
–ΩÇ≥Å––∞–ª—Å—∫–ª
–ΩÇ≥Å––∞–ª—Å—∫–ª
–ΩÇ≥Å∞–ª
–ΩÇ≥Åª—Ä–∏–π—Å–∫
–ΩÇ≥Åª—Ä–∏–π—Å–∫
–ΩÇ≥Åª—Ä–∏–π—Å–∫
–ΩÇ≥Åª—Ä–∏–π—Å–∫
–ΩÇ≥Åª—Ä–∏–π—Å–∫
–ΩÇ≥Åª—Ä–∏–π—Å–∫
–ΩÇ≥Åª—Ä—Ç—∞–∫
–ΩÇ≥Åª—Ä—Ç—∞–Ω–ª
–ΩÇ≥Åª—Ä—Ç—∞–Ω–ª
–ΩÇ≥Åª—Ä—Ç—∞–ΩÇ∏–Ω–ª
–ΩÇ≥Å∞–ª
–ΩÇ¥–∏–ª
–ΩÇ∞–¥—Ä–ª
–ΩÇ∞–¥—Ä–µ–∞–Ç–∏–∫
–ΩÇ∞–¥—Ä–µ–∞–Ω–∫
–ΩÇ∞–¥—Ä–µ–∞–Ω–∫
–ΩÇ∞–¥—Ä–µ–∞–Ω–∫
–ΩÇ∞–¥—Ä–µ–∞–Ω–∫
–ΩÇ∞–¥—Ä–µ–∞–Ω–∫
–ΩÇ∞–¥—Ä–µ–∞–ª
–ΩÇ∞–¥—Ä–ª
–ΩÇ∞–¥—Ä–Ç–∏–∫
–ΩÇ∞–¥—Ä–Ç–∏–∫
–ΩÇ∞–¥—Ä–Ç–∏–∫
–ΩÇ∞–¥—Ä–Ç–∏–∫
–ΩÇ∞–¥–ª
–ΩÇ¥–∏–ª
–ΩÇ∞–¥—Ä–ª
–ΩÇ∞–¥—é–à–ª
–ΩÇ∞–¥—é–à–±–∞–á—Ä–∏—Ü
–ΩÇ∞–¥—é–à–ª
–ΩÇ∞–¥—é–à–ª
–ΩÇ∞–¥—é–à–ª
–ΩÇ∞–¥—é–à–∫–ª
–ΩÇ∞–¥—é–à–∫–ª
–ΩÇ∞–¥—é–à–∫–ª
–ΩÇ∞–¥—é–à–ª
–Ω–ª
–Ω
–ø–∞–Ω–∫–≤–¥–æ—Ç
–∞–Ω–∫–≤–¥–æ—Ç
–∞–Ω—Ç—∫–ª
–Ω
–ø–∞–Ω–Å
–ø–∞–Ω–Å
–ø–∞–Ω–Å
–ø–∞–Ω–Å–Ω—å–∫
–∞–Ω–Å
–ø–∞–Ω–Å
–ø–∞
–ø–∞
–ø–∞
–ø–∞
–ø–∞
–ø–∞–Ç–∞–≤–µ—ø
–ø–∞–Ç–∏—ø
–ø–∞–Ç–∏—ø
–ø–∞–Ç–∏—ø—å–∫
–∞–Ç–∏—ø—å–∫
–∞–Ç–∏—ø–∫
–∞–Ç–æ–∞
–ø–∞–Ç–æ–∞
–ø–∞–Ç–∏—Ä–∞–∫—Ç
–∞–Ç–∏—Ä–∞–∫—Ç
–∞–Ñ–±–∏ª
–∞–¥
–∞–Ñ–±–Å
–ø–∞–Ñ–±–Å
–ø–∞–á–¥–É–Å
–ø–∞
–ø–∞
–ø–ø—Ä–∞—Ç
–ø–∏–∞—Ç–∏—Ç
–ø–∏–∞—Ç–∏—á–µ—Å–∫
–ø–∏–∞—Ç–∏—á–µ—Å–∫
–ø–∏–∞—Ç–∏—á–µ—Å–∫
–ø–∏–∞–∫
–ø–∏–∞–∫
–ø–––∞Çª—è–∫—Ü
–øÅ––∞–ª—Å—º–∏–Ω—Ä—Ç—∞–∫
–øÅ––∞–ª—Å—º–∏–∫
–øÅ∞Çæ–∞–¥–∏—Ä
–øÅ∞Çæ–∞–¥–∏—É
–≥–∞–ø–ªÅ∞Çæ–Ω
–≥–∞–ø–ø–—Å––∞––ΩÇ–∏—á–µ—Å–∫
–ø–ø–ø–—Å––∞–––∫
–ø–øÇµ–Ç–Ω–Ç–∫
–ø–øÇµ–Ç–Ω–Ç–Ω
–≥–∞–øÇµ–Ç–Ω–Ç–∫
–ø–∏—Ä–∞–Å—º–∏–∫
–ø–∞—Ä–µ–ª
–ø–∞—Ä–µ–ª
–ø–∞—Ä–µ–ª—Å—∫–ª
–ø–∞—Ä–µ–ª
–ø–Ç–ª–µ–∫
–ø–Ç–ª–µ∫
–∏–ª
–ø–Ç–ª–µ–∫
–ø–Ç–ª–µ–∫
–ø–Ç–ª–µ–∫
–ø–Ç–ª–∏—∏–∫
–ø–É—Ö–Ç–∏—∞–Å–∏—Ü
–∏–ª
–∏—Ä–≤—Ä–∏–π—Å–∫
–∏—Ä–ø—Å–∫
–∏—Ä–ø—Å–∫
–∏—∑–±—∑–∫
–∏—∑–±—∑–∫
–∏—≥–±—µ–º–µ–Ω—Ç
–∏–ª
–∏—≤–µ–¥—Ä–∞—Ç–æ—Ä
–∏—≤–µ–¥—Ä
–∏—≤–µ–¥—Ä
–∏—≤–µ—Ä
–∏—≤–µ—Ä
–∏—Ä
–∏—≤–∏—Å—Ü
–∏—≤–∏—Å–±–∞–Ω—Ç
–∏—≤–∏—Å–±–∞–Ω—Ç
–∏—≤–∏—Å–±–∞–Ω—Ç
–∏—≤–∏—Å–±–∞–Ω–π—Å–∫
–∏—≤–∏—Å–±–∞–Ω–π—Å–∫
–∏—≤–∏—Å–±–∞–Ω–∫
–∏—≤–∏—Å–Ä–æ–≤–∞
–∏—≤–∏—Å–Ä–æ—ã–æ–≤–∞
–∏—≤–∏—Å–∞
–∏—≤–∏—Å–∞
–∏—≤–∏—Å–∞
–∏–∞
–∏—∏—∞–É–à–∫–ª
—Ç—Ä–∏—Å–Ä–∫—Ç—Ä–∞—Ç–∏—á–µ—Å–∫
—Ç—Ä–∏—Å–Ä–∫—Ç—Ä–∞—Ç–∏—á–µ—Å–∫
—Ç—Ä–∏—Å–Ä–∫—Ç—Ä–∞—Ç–∏—á–µ—Å–∫
—Ç—Ä–∏—Å–Ä–∫—Ç—Ä–∞—Ç–∏—á–µ—Å–∫
—Ç—Ä–∏—Å–Ä–∫—Ç—Ä–∞—Ç–∏—á–µ—Å–∫
—Ç—Ä–∏—Å–Ä–∫—Ç—Ä–∞—Ç–∏—á–µ—Å–∫
—Ç—Ä–Ñ–ºÇµ–Ç–Ω—Å–∫
—Ç—Ä–Ñ–ºÇµ–Ç–Ω—Å–∫
—Ç—Ä–Ñ–ºÇµ–Ç–Ω—Å–∫
—Ç—Ä–à–∞
–∏–∞
–∏—∫ª
–∞–¥
–∏—∫ª
–∞—º–∏–∫
–∏—∫ª
–∞—º–∏–∫
–∏—∫ª
–∞–∫
–∏—∫ª
–∞–∫
–∏—∫–∫
–∏—∫–∫
–∏—µ–º–∏–π—Å–∫
–∏—º–∫
–∏—º–∫
–∏—º–∫
–∏—º–∫
–∏—º—è–∫–∫
–∏—º—è–∫–∫
–∏—º—è–∫–∫
–∏—º—è–á—Ä–à–∫–ª
—Ç—∫–æ–∞—Å–∞
–∏—∫–æ–∞—Å–∞
–∏—∫–æ–∞—Å–∏–∫
–∏—∫–æ–∞—Å–∏–∫
–∏—∫–æ–∞—Å–∏–∫
–∏—∫–æ–∞—Å–∫
–∏—∫–æ–∞—Å–∫
–∏–Ç–ª–ª–∫
–∏–Ç–ª–ª–Ω—Ω–∫
–∏–Ç–ª–ª–Ω—â–Ω—Å–∫
—Ç–Ç–ª–ª–Ω—â–Ω—Å–∫
—Ç–Ç–ª–ª–Ω—â–Ω—Å–∫
—Ç–Ç–ª–ª–Ω—â–Ω—Å–∫
—Ç–Ç–ª–º–∫
—Ç–Ç–ª–º–∫
—Ç–Ç–∏–∏—Å–∞
–∏–Ç–∏–∏—Å–∞
–∏–Ç–∏–∏—Å—Ç–∏—á–µ—Å–∫
—Ç–Ç–∏–∏—Å—Ç–∏—á–µ—Å–∫
—Ç–Ç–∏–∏—Å—Å–∫
—Ç–Ç–∏–∏—Å—Å–∫
—Ç–Ç–∏–∏—Å—Å–∫
—Ç–Ç–∏–∏—Å–∫
—Ç–Ç–∏–∏—Å–∫
—Ç–Ç–∫—É–∫
—Ç–Ñ–∫
—Ç–Ñ–∫
—Ç–Ñ–∫
—Ç–Ñ–∫
—Ç–Öª
–ΩÇ≥Å––∞–ª
—Ç–Öª
–ΩÇ≥Å––∞–ª—Å—∫–ª
—Ç–Öªµ–ø–ªÅæ–∞–≥
—Ç–Ö–ª
—Ç–Öª∏–∞–ª
—Ç–Öª∏–µ–º–¥–Ä–æ–ª
—Ç–Öª∏–Ç–ª–µ–∞—Ç–æ—Ä
–∏–Öª∏–Ç–ª–µ–∞—Ç–æ—Ä
–∏–Öª∏–Ç–ª–µ–∞—Ç–æ—Ä
–∏–Öª∏–Ç–ª–µ–∞–∫—É–∫
—Ç–à—º–∏–∫
–∏–à—º–∏–∫
–∏–à—º–∏–∫
–Å—≤–∏—Å—Ç–æ—Ä
–Å—≤–∏—Å—Ç–æ–à–∞
–Å—∫—Ç–µ—Ç
–∏—Å—Å–≥–ΩÇ
–Ü—Ç
–∏—Å—Å–≥–ΩÇ
–Ü—Ç
–∏—Å—æ–∫—Ü–
–Ü—Ç
–∏—Ç–º–∫
—∏—Å—Ç—Ä–Ö–ø–∞
–ø—∏—Å—Ç—Ä–Ö–ø–∞—Å—∫–ª
—∞—Å—Ç—Ä
—∏—Å—Ç—Ä–æ–ª
—∏—Å—Ç—Ä–æ–ª
—∏—Å—Ç—Ä–æ–ª
—∏—Å—Ç—Ä–æ–Ω–æ–º

–∞—Å–
–∞–∫
—Å–ª–±–∞–ª
—Å–ª–±–∞–∞
–ø—Å–ª–±–∞–∞
–ø—Å–ª–±–∞–∞
–ø—Å–ª–±–∞–∞
–ø—Å–º–∫–æ–Ñ–à–µ—Ä
—Å–º–∫–æ–Ñ–à–µ—Ä
—Å–Ω–æ–º
—Å–Ω–æ–º
—Å–∏—∏—∑–±–µ—Ç
–Ç–Ç–ª–∏—Å–±–µ—Ç
–Ç–Ç–ª–∏—Å–±–µ—Ç

—∞–É—¥ª∏–Ç–Ç–æ—Ä
–É–∞–∫—Ü–æ–Ω
–≥∫—Ü–æ–Ω
–≥∫—Ü–æ–Ω
∞–ª
—É–Å
–ø–Ñ–
–Ω–±–∞–ª
–Ñ–
–Ω–±–∞–ª
–Ñ–
–Ω–±–∞–ª
–Ñ–
–Ω–±–∞–ª
–Ñ–
–Ω–±–∞–Ω—µ–∞–Ç–∏–∫
–Ñ–
–Ω–±–∞–Ω—µ–∞–Ç–∏–∫
–Ñ–
–Ω–±–∞–Ω—µ–∞–Ç–∏–∫
–Ñ–Ä–à–∞
–Ñ–±—Ä–∏—Ü
–Ñ–±—Ä–∏—Ü
–Ñ–±—Ä–∏—Ü
–Ñ–±—æ–Å—º–∏–ª—é–à–∫–ª
–Ñ–±—æ–Å—º–∏–ª—é–à–∫–ª
–Ñ–Ñ–ª–µ–∞—
–Ü—Ç
–Ö–ª
—ÖÇ

—∞–Ö–ø–∞
–ø—ÖÇ

—∞–Ö–∏–ªΩ—–µ–∞–ª
–Ö–∏–ªΩ—–µ–∞–∑–æ–≤—Å–∫
–Ö–∏–ªΩ—–µ–∞–ª
–Ö–∏–ªΩ—–ª
–Ö–ºº
—Å–Ω–≤–ª
–Ö–Ω–Ω
∞–ª
—Ö–Ω–Ω
∞–ª
—Ö–Ω–Ω
∞–ª
—Ö–Ω–Ω–µ—Ç
–Ü–Ç–ª–µ–∫
—ç—Ç—Ä–ø–—Å∞–∫
—ç—Ç—Ä–ø–—Å∞–∫±
æ—Ç—Ä–ø–—
æ—Ç—Ä–ø–—
ª–µ–∫
—ç—Ç—Ä—
æ—Ç—Ä–ø–—
ª–µ–
æ—Ç—Ä–ø–—
ª–µ–
æ—Ç—Ä–ø–—
ª–µ–
æ—Ç—Ä